VideoWeb/VideoWeb/ClientApp/src/app/shared/footer/footer.component.scss

```diff
@@ -3,24 +3,37 @@
   flex-flow: column;
 }
 
-.govuk-footer__copyright-logo {
+.govuk-footer__copyright-logo,
+.govuk-footer__copyright-logo-scottish {
   display: inline-block;
   min-width: 125px;
   padding-top: 112px;
-  background-image: url('/assets/images/govuk-crest.png');
   background-repeat: no-repeat;
   background-position: 50% 0%;
   background-size: 125px 102px;
   text-align: center;
   text-decoration: none;
   white-space: nowrap;
 }
+
+.govuk-footer__copyright-logo {
+  background-image: url('/assets/images/govuk-crest.png');
+}
+
+.govuk-footer__copyright-logo-scottish {
+  background-image: url('/assets/images/scottish-crest-footer.png');
+}
```

Comment:
I think we should keep .govuk-footer__copyright-logo and always set the class to that. But have two new classes one of which being .govuk-footer__copyright-logo-scottish and something else for the other. And have that conditionally set with with the ngClass similar to what you've done

Reply:
yea sure, so I will add a class called  govuk-footer-background-uk and govuk-footer-background-scottish

Reply:
Would hmcts make more sense than uk? Or is it also hmcts in scotland?

Reply:
oh that makes sense, no for scottish its SCTS, I will rename it